homeassistant/components/metoffice/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
`test_form_cannot_connect`, `test_form_unknown_error`. should both finish, this way we test that the flow is able to recover from an error